modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -58,30 +60,10 @@ public class EmailNotificationTypeTest extends BaseNotificationTypeTest {
 
 	@Test
 	public void testSendNotification() throws Exception {
-		Assert.assertEquals(
-			0,
-			notificationQueueEntryLocalService.
-				getNotificationQueueEntriesCount());
 
-		NotificationTemplate notificationTemplate = _addNotificationTemplate(
-			false);
+		// One email including all main recipients
```

Comment:
@pedro-oliveira446 sort these blocks, the block:

// One email including all main recipients

should be after:

// Multiples emails for each main recipients, with terms and semicolon separator